### DISCUSSION TITLE
Border around selected module

### DISCUSSION THREAD

**Discussion by alltiagocom** [DISTRHO/Cardinal#510] [Ideas]
It would be great if a selected module would have a border around it so when we want to delete it, we know exactly what is selected. Or even mimic when 2 or more are selected with the border and the red overlay. 
I noticed that if I hold Shift, it does that, but it would be great if just clicking it would do the same thing.

**Comment by dromer** [comment]
Such behavior ultimately comes from VCV Rack.
It would probably be good to make this request on the VCV forums so that, if implemented, it "trickles down" to Cardinal as well.

**Reply by alltiagocom** [reply]
I see. I'm not very familiar with how these things work, to be honest, so I thought it would be possible for the developers of Cardinal to implement something custom, even if VCV Rack doesn't work that way. I mean, the SHIFT+click is ok, but it would be more visual even if we don't hold shift.

Now that you mention the VCV Rack forums, let me ask you an off-topic question, since I'm new to Cardinal: is it ok to ask questions that involve Cardinal, on those forums? Are you aware of people being "not cool" with it or something? It seems that it is a very active community and I would like to ask questions and all that, but I don't want to be talking about another piece of software there, even if it's based on VCV Rack. Hope it makes sense?

**Reply by falkTX** [reply]
VCV forums are for VCV things, so it is not the most appropriate to discuss forks in there.
They even have a written rule that discussion regarding commercial forks of Rack is forbidden. Cardinal is not commercial in nature, but still I always been careful to not mention it directly.

for asking things, one can simply compare and check how the official VCV Rack free does things, since it will pretty much match Cardinal in most areas.

regarding the custom Cardinal behaviour, we could indeed handle things differently, but for maintenance reasons I prefer to stick with the original behaviour as much as possible. the exceptions appear when handling the entire project as a single self-contained plugin, otherwise we follow Rack

**Reply by alltiagocom** [reply]
Thanks @falkTX 
Yeah, since I'm not familiar with the technicalities on how "forks" work and all that, it seems to be something more complex that I can even grasp, but yeah, this would be good to have originally in VCV Rack so maybe I can suggest it in the forum and hopefully that will be implemented so then Cardinal can use that feature as well. Thanks for clarifying that and the issue with the forums as well. I would assume that discussing Cardinal there wouldn't be the best option, but decided to ask anyway just in case.

**Reply by falkTX** [reply]
the best course of action is to use VCV Rack free for a bit, and talk about it there.

do note that user requests (and even developer ones) are not really taken serious on their forums. it is treated more of a community thing, a place for folks to discuss random stuff, best to not expect many official VCV replies in there..

I am considering setting up a forum to discuss Cardinal and other DISTRHO and KXStudio things, but there are other way more important priorities at the moment. And in this case it doesnt help the situation.. just thought to mention, we might see it sometime in April, no promises

**Reply by alltiagocom** [reply]
Yeah a forum for Cardinal would be awesome indeed! 
I will take into account what you just said about the VCV Rack forum. Thanks.

So another Portuguese fellow here, huh? (sim, também sou mais um "tuga" haha). It's great to see that Cardinal is being developed and maintained "in the house"! 😊
